src/index.ts

```diff
@@ -48,6 +48,8 @@ export interface PostcssThemeOptions {
   defaultTheme?: string;
   /** Transform CSS variable names similar to CSS-Modules */
   modules?: string | ScopedNameFunction;
+  /** Determines if we want to use all themes or individual themes */
+  forceSingleTheme?: boolean;
```

Comment:
Can you document this in the Readme, and provide an example of a postcss config that outputs multiple files?